tests/test_pex.py

```diff
@@ -373,3 +374,29 @@ def test_pex_run_custom_setuptools_useable():
       )
       rc = PEX(pex.path()).run()
       assert rc == 0
+
+
+def test_pex_run_conflicting_custom_setuptools_useable():
+  # Here we use an older setuptools to build the pex which has a newer setuptools requirement.
```

Comment:
Are we testing these specific setuptool versions, or more generally setuptools conflicting? If someone changed from 40.4.3 to 44.2.1 in a year, would that be testing something different?

Would be helpful to add a quick note saying "These specific version numbers matter due to API changes" etc. vs "The particular version numbers do not matter, rather simply having different versions matters."

Reply:
I was trying to point out these specific versions numbers by including the comment diff. IE: these two specific versions work for the test since the exhibit the illustrated API diff.

Reply:
Okay makes sense. Consider leading with a comment like "Setuptools changed one of its APIs in version x." Then the rest you have below and diff. 

It's fine as is if you don't want to change, but might make things even more clear.

Reply:
OK - I'm not even sure what version it changed in! I just took a low and a high from the pex supported range. I think I'll leave as is.